CHANGES.md

```diff
@@ -0,0 +1,373 @@
+ - **st_b0shim**: Fix a bug when scanner constraints are not implemented. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/543)
+ - **st_b0shim, st_mask, st_prepare_fieldmap**: Fix st_b0shim realtime-dynamic order 0 and output folder for masking and fieldmap. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/353)
+ - **st_check_dependencies**: Change command that checks for sct installation. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/298)
+ - **st_check_dependencies**: Change SCT CLI call to check version. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/402)
+ - **st_dicom_to_nifti**: Fix phasediff renaming issue for dual-echo fieldmaps. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/447)
+ - **st_dicom_to_nifti**: Update dcm2bids to version 3.0.1. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/470)
+ - **st_dicom_to_nifti**: Improve reliability of phase diff renaming. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/504)
+ - **st_dicom_to_nifti**: Solve dcm2niix not in PATH issue and improve st_prepare_fieldmap documentation. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/523)
+ - **st_dicom_to_nifti, st_prepare_fieldmap**: Bug fix related to updating FSLeyes version and Shimming Toolbox version. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/483)
+ - **st_download_data**: Fix broken link in st_download_data data_create_coil_profiles. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/535)
+ - **st_prepare_fieldmap**: Field mapping pipeline improvement. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/410)
+ - **st_prepare_fieldmap**: Resolve prelude error if --savemask directory is not already created when field mapping. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/448)
+ - **st_prepare_fieldmap**: Change phase range in field mapping calculations. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/499)
+ - **st_realtime_shim**: Fix realtime shimming mask resampling modes. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/289)
+ - **st_realtime_shim**: gradient realtime shim output coordinate system. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/310)
+ - **st_realtime_shim**: Gradient realtime_shim orientation and nan problems. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/323)
+ - Solve launchers needing to be launched from the ST venv. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/346)
+ - Restrict numpy version < 2.x. [View pull request](https://github.com/shimming-toolbox/shimming-toolbox/pull/527)
```

Comment:
The signal recovery PR was not included in the changelog because it was missing a label. I added it back. It appears in the "feature" section though. 

Also, the changes are not exactly in chronological order, they are in order of CLIs and PR numbers so it can be confusing when looking at PRs that occurred over 3 years. If we have releases that occur more frequently, it should be less confusing.

Reply:
But shouldn't we remove changes that are not applied in the current state of the code?

Reply:
Since it got removed and added back in the same version, I think it makes sense. I removed it in [ba97c7e](https://github.com/shimming-toolbox/shimming-toolbox/pull/541/commits/ba97c7e18aba3f16c276d9e739b23dd073e82bae)